SPECS-EXTENDED/libucil/libucil.spec

```
@@ -0,0 +1,217 @@
+Summary:        Library to render text and graphic overlays onto video images
+Name:           libucil
+Version:        0.9.10
+Release:        22%{?dist}
+License:        GPLv2+
```

Nit: could be an SPDX expression

```suggestion
License:        GPL-2.0-or-later
```